akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/TestProbe.scala

```diff
@@ -220,6 +220,11 @@ abstract class TestProbe[M] {
    */
   @InternalApi protected def expectMessageClass_internal[C](max: FiniteDuration, c: Class[C]): C
 
+  /**
+   * Receive one message of type `M` within `remainingOrDefault` as the `TestTimeFactor`.
+   */
+  def receiveOne(): M
+
   /**
    * Same as `receiveN(n, remaining)` but correctly taking into account
    * the timeFactor.
```

Comment:
I know this doc about "correctly taking into account the timeFactor" is sprinkled all over the place, but I think it's strange. I'd say "but using the default timeout as deadline". 

`timeFactor` has to do with `dilation` and this doc makes it sound as we do something wrong in the other method.

Reply:
👍 updated in both DSL TestProbes